### ISSUE TITLE
Error in validation index pages

### ISSUE THREAD

**Issue by pimbongaerts** [pimbongaerts/mesophotic#14]
Validators getting a 500 Internal Server Error when trying to navigate to pages 2-5 on Publications page (editor/admin version).

Production log:
```
2017-09-21T14:06:58.217428 #31391]  INFO -- : Started GET "/publications?page=2&search_params%5Bfields%5D%5B%5D=title&search_params%5Bfields%5D%5B%5D=abstract&search_params%5Bfields%5D%5B%5D=contents&search_params%5Bfields%5D%5B%5D=authors&search_params%5Bpublication_formats%5D%5B%5D=article&search_params%5Bpublication_formats%5D%5B%5D=review&search_params%5Bpublication_formats%5D%5B%5D=report&search_params%5Bpublication_formats%5D%5B%5D=thesis&search_params%5Bpublication_formats%5D%5B%5D=book&search_params%5Bpublication_formats%5D%5B%5D=chapter&search_params%5Bpublication_types%5D%5B%5D=scientific&search_params%5Bpublication_types%5D%5B%5D=technical&search_params%5Bpublication_types%5D%5B%5D=popular" for 124.150.89.28 at 2017-09-21 14:06:58 -0700
I, [2017-09-21T14:06:58.219054 #31391]  INFO -- : Processing by PublicationsController#index as HTML
I, [2017-09-21T14:06:58.219135 #31391]  INFO -- :   Parameters: {"page"=>"2", "search_params"=>{"fields"=>["title", "abstract", "contents", "authors"], "publication_formats"=>["article", "review", "report", "thesis", "book", "chapter"], "publication_types"=>["scientific", "technical", "popular"]}}
I, [2017-09-21T14:06:58.220266 #31391]  INFO -- : Completed 500 Internal Server Error in 1ms
F, [2017-09-21T14:06:58.221924 #31391] FATAL -- : 
NoMethodError (undefined method `keys' for ["title", "abstract", "contents", "authors"]:Array):
  app/controllers/publications_controller.rb:213:in `block in search_params'
  app/controllers/publications_controller.rb:213:in `reduce'
  app/controllers/publications_controller.rb:213:in `search_params'
  app/controllers/publications_controller.rb:18:in  @`index'
```

**Comment by ryanbooker** [comment]
Bugger. Ok looking into it.

**Comment by ryanbooker** [comment]
Fixed by f3a99cafbea03c0449760a52ceaa856f3986ff91
